### ISSUE TITLE
Reader

### ISSUE THREAD

**Issue by nblumhardt** [serilog/serilog-formatting-compact#11]
I've put together a basic reader for the (non-rendered) compact JSON format here:

https://github.com/nblumhardt/serilog-formatting-compact-reader

There would be some benefits maintaining this alongside the formatters, but also more effort in making the implementation complete and supportable. (In any case it would remain a separate package, as there's a JSON.NET dependency in there.)

Raising this for discussion/tracking.

**Comment by aaronpowell** [comment]
Funnily enough I actually wrote something similar to this in the past week, although a bit more brutal. Our use case is that we've got a series of microservices in which we're pushing the logs into RabbitMQ (via the appropriate sink) and then have a queue monitor running elsewhere that periodically drains the RabbitMQ queue and pushes the logs elsewhere.

It's kind of cumbersome at the moment how we consume the data coming out of RabbitMQ, some custom parsing and such, but I can see this being useful for us to then dump the logs into longer term storage locations.
